src/main/java/io/github/jhipster/registry/security/jwt/TokenProvider.java

```diff
@@ -97,7 +94,7 @@ public Authentication getAuthentication(String token) {
 
     public boolean validateToken(String authToken) {
         try {
-            Jwts.parser().setSigningKey(key).parseClaimsJws(authToken);
+        	Jwts.parserBuilder().setSigningKey(key).build().parseClaimsJws(authToken);
```

Comment:
indent

Reply:
will the PR be merged automatically? i see you approved the changes but the branch is not yet merged

Reply:
it's good :)